common/core/web/keyboard-processor/src/build.sh

```diff
@@ -6,14 +6,18 @@
 ## START STANDARD BUILD SCRIPT INCLUDE
 # adjust relative paths as necessary
 THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
-. "$(dirname "$THIS_SCRIPT")/../../resources/build/build-utils.sh"
+KEYMAN_ROOT="$(dirname "$THIS_SCRIPT")/../../../../.."
```

Comment:
I'd prefer to leave the standard build script include as similar as possible in all scripts -- makes it easier to maintain later if we have to change it for some reason. `KEYMAN_ROOT` should be set by build-utils.sh so you can use it for subsequent includes.